.github/CODEOWNERS

```diff
@@ -675,23 +675,143 @@ playwright.storybook.config.ts @grafana/grafana-frontend-platform
+/public/app/core/services/theme.ts @grafana/grafana-frontend-platform
+/public/app/core/specs/backend_srv.test.ts @grafana/grafana-frontend-platform
+/public/app/core/specs/factors.test.ts @grafana/dashboards-squad
+/public/app/core/specs/flatten.test.ts @grafana/grafana-frontend-platform
+/public/app/core/specs/impression_srv.test.ts @grafana/grafana-frontend-platform
+/public/app/core/specs/rangeutil.test.ts @grafana/grafana-frontend-platform
+/public/app/core/specs/store.test.ts @grafana/grafana-frontend-platform
+/public/app/core/specs/TableModel.test.ts @grafana/partner-datasources
+/public/app/core/specs/ticks.test.ts @grafana/grafana-plugins-platform-frontend
+/public/app/core/specs/time_series.test.ts @grafana/dataviz-squad
+/public/app/core/store.ts @grafana/grafana-frontend-platform
+/public/app/core/TableModel.ts @grafana/partner-datasources
+/public/app/core/time_series2.ts @grafana/dataviz-squad
+/public/app/core/trustedTypePolicies.ts @grafana/grafana-frontend-platform
+/public/app/core/utils/accessControl.ts @grafana/identity-access-team
+/public/app/core/utils/applyStateChanges.ts @grafana/dashboards-squad
+/public/app/core/utils/arrayMove.ts @grafana/grafana-frontend-platform
+/public/app/core/utils/auth.ts @grafana/identity-access-team
+/public/app/core/utils/browser* @grafana/grafana-frontend-platform
+/public/app/core/utils/colors.ts @grafana/grafana-frontend-platform
+/public/app/core/utils/ConfigProvider.tsx @grafana/grafana-frontend-platform
+/public/app/core/utils/CorsSharedWorker.ts @grafana/grafana-frontend-platform
+/public/app/core/utils/CorsWorker.ts @grafana/grafana-frontend-platform
+/public/app/core/utils/connectWithReduxStore.tsx @grafana/grafana-frontend-platform
+/public/app/core/utils/dag* @grafana/grafana-frontend-platform
+/public/app/core/utils/deferred.ts @grafana/grafana-frontend-platform
+/public/app/core/utils/docsLinks.ts @grafana/grafana-frontend-platform
+/public/app/core/utils/errors* @grafana/grafana-frontend-platform
+/public/app/core/utils/explore* @grafana/observability-traces-and-profiling
+/public/app/core/utils/factors.ts @grafana/dashboards-squad
+/public/app/core/utils/fetch* @grafana/grafana-frontend-platform
+/public/app/core/utils/flatten.ts @grafana/grafana-frontend-platform
+/public/app/core/utils/isShallowEqual.ts @grafana/grafana-frontend-platform
+/public/app/core/utils/kbn* @grafana/grafana-frontend-platform
+/public/app/core/utils/navBarItem-translations.ts @grafana/grafana-search-navigate-organise
+/public/app/core/utils/object* @grafana/grafana-frontend-platform
+/public/app/core/utils/query* @grafana/grafana-datasources-core-services
+/public/app/core/utils/richHistory* @grafana/observability-traces-and-profiling
+/public/app/core/utils/set.ts @grafana/grafana-frontend-platform
+/public/app/core/utils/shortLinks* @grafana/sharing-squad
+/public/app/core/utils/ticks.ts @grafana/grafana-plugins-platform-frontend
```

Comment:
not 100% sure this is a plugins-platform thing.

Reply:
it's a no one thing 😄 no one uses it. the only place it's still exposed is via `sharedDependencies`, so i figure you guys would be the best to determine when we can kill it for good 🙏

Reply:
`@grafana/plugins-platform-frontend` or `@grafana/grafana-plugins-platform-frontend`?

Reply:
the former - fixed it 👍 thanks for spotting 🙇